src/app/home/home.component.scss

```diff
@@ -1,9 +1,50 @@
-.x-small small {
-    font-size: 60%;
+$debug-pf-border:0px;
+
+$content-offset: 15px;
+
+$debug-pf-color: purple;
+$debug-pf-row-color:lighten($debug-pf-color,25%);
+$debug-pf-column-color:lighten($debug-pf-color,50%);
+
+
+.container-cards-pf
+{
+  position:relative;
+  border :solid $debug-pf-border $debug-pf-color;
+  width:100%;
+  margin-top:0px;
+  overflow:auto;
+}
+.row
+{
+  padding-top:$content-offset/2;
+  position:relative;
+  border:solid $debug-pf-border $debug-pf-row-color;
 }
 
-.pull-bottom {
-  display: table-cell;
-  vertical-align: bottom;
-  float: none;
+div[class*='col-']
+{
+  position:relative;
+  border:solid $debug-pf-border $debug-pf-column-color;
+  padding-left:$content-offset;
+  padding-right:$content-offset;
+  .card-pf
+  {
+    padding:0px $content-offset;
```

Comment:
this makes sense to me... going forward , for consistency sake i will not use px for 0